frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -23,7 +37,7 @@
         </b-button>
         <br />
         <b-button
-          class="p-2 mt-3"
+          class="p-2 test-delete-link"
```

Comment:
are you sure you want this name on the element? Why not find the button as before?

Reply:
weil mich es stört das wenn man die button position verändert der gesamte test einem um die ohren fliegt. weil at() nicht mehr passt